src/validation_workflow/validation_rpm.py

```diff
@@ -0,0 +1,28 @@
+# Copyright OpenSearch Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+
+from system.temporary_directory import TemporaryDirectory
+from validation_workflow.download_utils import DownloadUtils
+from validation_workflow.validation import Validation
+
+
+class ValidationRpm(Validation, DownloadUtils):
+    tmp_dir = TemporaryDirectory()
```

Comment:
If this is a temp dir that all the test requires, why not just put it back to the parent class?